source/tutorials/docker.rst

```diff
@@ -56,18 +56,11 @@ packages to run Docker images as containers.
 
       sudo swupd bundle-add containers-basic
 
-#. Start the Docker daemon through systemd manager by running this command:
+#. Enable and start the Docker daemon through systemd manager by running this command:
 
    .. code-block:: bash
 
-      sudo systemctl start docker
-
-   If you want Docker to start automatically on boot, enable the
-   systemd service by running this command:
-
-   .. code-block:: bash
-
-      sudo systemctl enable docker
+      sudo systemctl enable --now docker
```

Comment:
I'm offering a third alternative in my recommendation above. I hope you continue to contribute. 
If you like, you can follow my recommendation.  If not, we'll leave the document as it stands.

Reply:
Whatever we choose, there will always be some audience that asks about the opposite usage.
I like @donnydavis 's suggestion as the primary method whenever there is a systemctl start + enable.

Reply:
I've asked other Clear Linux engineers' opinions on this, and they want the bar lowered for entry-level Docker users. I offer a compromise, adding the combo `start-enable` as an alternative--without deleting the first original commands. I fully understand that advanced Docker users would prefer the combo, but that misses the point.

Reply:
IMO that kinda defies logic - why would you want beginners starting a daemon they would likely expect to be running on reboot and not just do it the right way to begin with. 

Confusing people with too many options is also not a good idea, because you are right. Beginners may not understand. 

If the option is to make a more confusing doc by presenting too many ways to start and enable to the daemon, then just leave it how it is.